tests/backup/backup_restore_basic_test.go

```diff
@@ -1932,18 +1933,21 @@ var _ = Describe("{MultipleInPlaceRestoreSameTime}", func() {
 			log.InfoD("Taking backup of application for different combination of restores")
 			var sem = make(chan struct{}, 10)
 			var wg sync.WaitGroup
+			var mutex sync.Mutex
 			for _, bkpNameSpace := range bkpNamespaces {
+				time.Sleep(timeBetweenConsecutiveOps)
 				sem <- struct{}{}
-				backupName = fmt.Sprintf("%s-%s-%v", BackupNamePrefix, bkpNameSpace, RandomString(5))
 				wg.Add(1)
+				backupName = fmt.Sprintf("%s-%s-%v", BackupNamePrefix, bkpNameSpace, RandomString(5))
 				go func(bkpNameSpace string, backupName string) {
 					defer GinkgoRecover()
 					defer wg.Done()
-					defer func() { <-sem }()
```

Comment:
I do not think there would be any use for the variable 'sem' if we remove this.

Reply:
Make sense let me add it back.